src/components/EventItem.jsx

```diff
@@ -1,17 +1,27 @@
 import { requestUser } from "@/lib/fetchAPI";
 import { EventItemAuthor } from "./EventItemAuthor";
 import Link from "next/link";
-import { Image } from "next/image";
+import Image from "next/image";
 
 export const EventItem = async ({ events }) => {
-  console.log(events.image);
   return (
     <div className="card card-compact bg-base-100 shadow-lg">
-      <div className="h-32 bg-cover bg-center rounded-t-xl"></div>
+      <div className="h-40 max-h-[500px] bg-cover bg-center rounded-t-xl">
+        <Image
+          src={events.image}
```

Comment:
does adding the "unoptimized" props in <Image/> without adding the remotepattern in nextjs.config works?